ci/gpu/build.sh

```diff
@@ -54,6 +54,19 @@ conda info
 conda config --show-sources
 conda list --show-channel-urls
 
+################################################################################
+# TEST - C++
+################################################################################
+
+cd "$WORKSPACE/cpp/examples/downstream"
+
+gpuci_logger "Build downstream C++ example"
+mkdir build
+cd build
+cmake ..
+make
```

Comment:
```suggestion
cmake --build .
```
So these scripts don't break if the ci env change the default generator to ninja